Lesson3/src/Components/Menu.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import MenuItem from './menuItem';
+
+export default class Menu extends React.Component {
+  render() {
+
+    const items = this
```

Comment:
```suggestion
    const items = this.props.items.map(...
```
такой практики в чужом коде я еще не видел, может ей и есть место быть, но код сложнее читать:)

Это не касается конструкций вида
```
fetch(...)
.then(....)
.catch(...)
```

Reply:
Эксперименты с экстеншенами для VSCode, которые форматируют код)